build-logic/convention/src/main/kotlin/ext/ComposeExt.kt

```diff
@@ -20,6 +20,8 @@ import org.jetbrains.kotlin.compose.compiler.gradle.ComposeCompilerGradlePluginE
 
 fun Project.configureCompose() {
     extensions.configure<ComposeCompilerGradlePluginExtension> {
-        stabilityConfigurationFile.set(rootProject.file("stability_config.conf"))
+        stabilityConfigurationFiles.addAll(
+            rootProject.layout.projectDirectory.file("compose_compiler_config.conf"),
```

Comment:
Will this still include the `stability_config.conf` file in the project?

Reply:
Sorry, there was a typo for the file name, fixed in #374.